docs/actions.rst

```diff
@@ -360,31 +359,57 @@ When a rule's condition(s) are satisfied we get the results back as:
 
 debug
 *****
-  Write the event to stdout
-  It accepts an arbitrary dict that it's injected into the event to be printed
-
-none
-****
-  No action, useful when writing tests
-  No arguments needed
-
-echo
-****
-.. list-table:: Write a user specified message to stdout
+.. list-table:: debug ansible-rulebook
    :widths: 25 150 10
    :header-rows: 1
 
    * - Name
      - Description
      - Required
-   * - message
-     - A terse message to display to stdout
-     - Yes
+   * - msg
+     - A simple string or an array of strings, which can have references to event or events
+     - No
+   * - var
+     - The variable to print, which can have references to event or events. Using {{ }} is optional.
```

Comment:
As far as I can see in the code, curly brackets are not supported here.

Reply:
@Alex-Izquierdo Every action args goes thru a substitution first and anything that is left is handled in the var for a debug.